ruby/test/Ice/properties/Client.rb

```diff
@@ -109,5 +109,20 @@ def run(args)
         end
         puts "ok"
 
+        print "testing Ice.initialize with args... "
+        args = ["--Foo.Bar=1", "--Foo.Bar=2", "--Ice.Trace.Network=3"]
+        Ice.initialize(args) do |communicator|
+            test(communicator.getProperties().getIceProperty("Ice.Trace.Network") == "3")
+            test(args.length == 2)
+        end
+
+        args = ["--Foo.Bar=1", "--Foo.Bar=2", "--Ice.Trace.Network=3"]
+        Ice.initialize(args) do |communicator, remArgs|
```

Comment:
I don't understand why we provide this remArgs syntax since we already modify args in place. Should we remove it?